specification/inference/_types/CommonTypes.ts

```diff
@@ -585,6 +585,8 @@ export class CohereServiceSettings {
   api_key: string
   /**
    * For a `text_embedding` task, the types of embeddings you want to get back.
+   * Use `binary` for binary embeddings, which are encoded as bytes with signed int8 precision.
+   * Use `bit` for binary embeddings, which are encoded as bytes with signed int8 precision (this is a synonym of `binary`).
```

Comment:
You'll also want to add these two new values to the `enum CohereEmbeddingType` list a bit further down in the file so that they are treated as valid options in code generated from the spec.